diagnostic.md

```diff
@@ -24,6 +38,9 @@ the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
 
+$('li < ol')
+$('ol > .listerItem)
```

Comment:
You're thinking about it like CSS. it's something like:
`$('listerItem').parent().siblings()`